torch/jit/__init__.py

```diff
@@ -789,6 +790,95 @@ def wrap_check_inputs(check_inputs):
 
     return [{'forward' : c} for c in check_inputs]
 
+
+def freeze(mod, preserved_attrs : Optional[List[str]] = None):
+    r"""
+    Freezing a :class:`ScriptModule` will clone it and attempt to inline the cloned
+    module's submodules, parameters, and attributes as constants
+    in the TorchScript IR Graph. By default, `forward` will be preserved,
+    as well as attributes specified in `preserved_attrs`. Additionally, attributes which
+    are modified in the `forward` or in a method specified in `preserved_attrs`
+    will be preserved.
```

Comment:
I think the second sentence could be worded more simply as "Additionally, any attribute that is modified within a preserved method will be preserved`.